heron/healthm/src/java/BUILD

```diff
@@ -0,0 +1,71 @@
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+healthmgr_deps_files = [
+    "//heron/api/src/java:api-java",
+    "//heron/api/src/java:classification",
+    "//heron/common/src/java:basics-java",
+    "//heron/common/src/java:config-java",
+    "//heron/common/src/java:utils-java",
+    "//heron/spi/src/java:common-spi-java",
+    "//heron/spi/src/java:packing-spi-java",
+    "//heron/spi/src/java:statemgr-spi-java",
+    "//heron/spi/src/java:utils-spi-java",
+    "//heron/spi/src/java:metricsmgr-spi-java",
+    "//heron/proto:proto_common_java",
+    "//heron/proto:proto_metrics_java",
+    "//heron/proto:proto_packing_plan_java",
+    "//heron/proto:proto_scheduler_java",
+    "//heron/proto:proto_tmaster_java",
+    "//heron/proto:proto_topology_java",
+    "//heron/metricsmgr/src/thrift:thrift_scribe_java",
+    "//heron/scheduler-core/src/java:scheduler-java",
+    "//third_party/java:jackson",
+    "@com_google_protobuf_protobuf_java//jar",
+    "@commons_cli_commons_cli//jar",
+    "@com_fasterxml_jackson_jaxrs_jackson_jaxrs_json_provider//jar",
+    "@com_fasterxml_jackson_jaxrs_jackson_jaxrs_base//jar",
+    "@com_fasterxml_jackson_module_jackson_module_jaxb_annotations//jar",
+    "@com_jayway_jsonpath//jar",
+    "@net_minidev_json_smart//jar",
+    "@javax_ws_rs_javax_ws_rs_api//jar",
+    "@javax_annotation_javax_annotation_api//jar",
+    "@javax_inject_javax_inject//jar",
+    "@org_apache_thrift_libthrift//jar",
+    "@org_glassfish_jersey_core_jersey_client//jar",
+    "@org_glassfish_jersey_core_jersey_common//jar",
+    "@org_glassfish_jersey_ext_jersey_entity_filtering//jar",
+    "@org_glassfish_jersey_media_jersey_media_json_jackson//jar",
+    "@org_glassfish_jersey_bundles_repackaged_jersey_guava//jar",
+    "@org_glassfish_hk2_hk2_api//jar",
+    "@org_glassfish_hk2_hk2_utils//jar",
+    "@org_glassfish_hk2_hk2_locator//jar",
```

Comment:
@kramasamy As of now Dhalion uses http client to query tracker and fetch metrics. These dependencies are related to the client and parsing the json returned by the tracker.